dotcom-rendering/src/web/components/LiveBlockContainer.tsx

```diff
@@ -0,0 +1,194 @@
+import { css } from '@emotion/react';
+import type { ArticleFormat } from '@guardian/libs';
+import { isString } from '@guardian/libs';
+import {
+	body,
+	from,
+	headline,
+	neutral,
+	space,
+} from '@guardian/source-foundations';
+import { decidePalette } from '../lib/decidePalette';
+import { FirstPublished } from './FirstPublished';
+
+type BlockContributor = {
+	name: string;
+	imageUrl?: string;
+	largeImageUrl?: string;
+};
+
+type Props = {
+	id: string;
+	children: React.ReactNode;
+	format: ArticleFormat;
+	blockTitle?: string;
+	blockFirstPublished?: number;
+	blockFirstPublishedDisplay?: string;
+	blockId: string;
+	isLiveUpdate?: boolean;
+	contributors?: BlockContributor[];
+	isPinnedPost: boolean;
+	supportsDarkMode: boolean;
+	isOriginalPinnedPost?: boolean;
+	host?: string;
+	pageId?: string;
+};
+
+const LEFT_MARGIN_DESKTOP = 60;
+const SIDE_MARGIN = space[5];
+const SIDE_MARGIN_MOBILE = 10;
+
+const Header = ({ children }: { children: React.ReactNode }) => {
+	return (
+		<header
+			css={css`
+				padding-right: ${space[3]}px;
+				display: flex;
+				flex-direction: column;
+			`}
+		>
+			{children}
+		</header>
+	);
+};
+
+const BlockTitle = ({ title }: { title: string }) => {
+	return (
+		<h2
+			css={css`
+				${headline.xxsmall({ fontWeight: 'bold' })}
+				margin-bottom: ${space[2]}px;
+			`}
+		>
+			{title}
+		</h2>
+	);
+};
+
+const BlockByline = ({
+	name,
+	format,
+	imageUrl,
+}: {
+	name: string;
+	format: ArticleFormat;
+	imageUrl?: string;
+}) => {
+	const palette = decidePalette(format);
+	return (
+		<div
+			css={css`
+				display: flex;
+				flex-direction: row;
+				padding-bottom: ${space[1]}px;
+			`}
+		>
+			{!!imageUrl && (
+				<div style={{ width: '2.25rem', height: '2.25rem' }}>
+					<img
+						src={imageUrl}
+						alt={name}
+						css={css`
+							border-radius: 100%;
+							object-fit: cover;
+							height: 2.25rem;
+							width: 2.25rem;
```

Comment:
```suggestion
							height: ${space[9]}px;
							width: ${space[9]}px;
```

Reply:
These were redundant with the container’s width